federation/src/typeorm/repository/User.ts

```diff
@@ -0,0 +1,67 @@
+import SearchUsersFilters from '@/graphql/arg/SearchUsersFilters'
```

Comment:
remove

Reply:
it is a copy from backend modul, so why should it be removed here?